lib/fibonacci.rb

```diff
@@ -5,8 +5,19 @@
 # ....
 # e.g. 6th fibonacci number is 8
 
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) with n being the input.
```

Comment:
Actually your solution is O(2^n) in time complexity.  We'll talk about this in the recursion lesson.